nvtabular/loader/tensorflow.py

```diff
@@ -297,26 +302,40 @@ def _LONG_DTYPE(self):
     def _FLOAT32_DTYPE(self):
         return tf.float32
 
+    def _to_dlpack(self, gdf):
```

Comment:
This method doesn't always return a dlpack object - and I'm wondering if we should rename. Maybe something like ```def _pack(self, df)``` for this method and ```def _unpack(self, df)``` for the _from_dlpack method?

Also I'd like to stop using 'gdf' for cases where the dataframe can be either a pandas or cudf dataframe. Can we go with 'df' here instead?